### DISCUSSION TITLE
알바폼 location

### DISCUSSION THREAD

**Discussion by gayeong0916** [team-final/albaform#173] [General]
![화면 캡처 2024-10-07 004635](https://github.com/user-attachments/assets/9d5265ae-213c-42a7-a1aa-7c46e5c7ba4a)

이 부분을 운영 Q&A를 통해서 물어봤는데
location 을 통해서 가지고 오라던데

서울특별시, 부산광역시, 경상남도 등등 여러가지의 경우의 수가 있는데 이걸 어떻게 끊어서 보여줘야할지.. 더 좋은 방법이 어떤건지 잘 모르겠습니다 ㅠ

**Comment by clyde-yoonjae** [comment]
https://ko.wikipedia.org/wiki/%EB%8C%80%ED%95%9C%EB%AF%BC%EA%B5%AD%EC%9D%98_%ED%96%89%EC%A0%95_%EA%B5%AC%EC%97%AD#%EC%B6%A9%EC%B2%AD%EB%B6%81%EB%8F%84

흠....

**Comment by clyde-yoonjae** [comment]
딱알았다

**Comment by clyde-yoonjae** [comment]
두번쨰 공백 앞에서짜르시면됩니다.

**Reply by gayeong0916** [reply]
오 굿! 굉장히 간단한 거였네요

**Comment by JuhyeokC** [comment]
```js
const locationSplit = location.split(' ') // [ '서울특별시', '종로구', '남은', '주소' ]
```
해서 배열의 0 1 번 쓰는?
애초에 간략한 주소를 같이 저장해두면 좋을 것 같긴한데..
